resources/lang/fr/flights.php

```diff
@@ -17,18 +17,18 @@
     'arr'              => 'Arr',
     'level'            => 'Niveau',
     'alternateairport' => 'Aéroport Alternatif',
-    'inbound'          => 'Vols à l\'arrivée',
-    'outbound'         => 'Vols au départ',
-    'none'             => 'Auncun vols trouvé.',
+    'inbound'          => 'Vols entrants',
+    'outbound'         => 'Vols sortants',
```

Comment:
Not sure here, "entrats/sortants" is syntactically good but I think "à l'arrivée/au départ" is more meaningful

Reply:
Il me semble que vols entrants et vols sortants est affiché en titre dans les aéroports.